cockroachdb/README.md

```diff
@@ -0,0 +1,3 @@
+# Package `cockroachdb`
```

Comment:
From what I understand, this package has nothing to do with CockroachDB but with pgx, hasn't it? 

We have a package `mongo` for MongoDB related thing, I would name this package `postgresql`.